rethinkdb/tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py38
+envlist =
+    py{27,38}-{2.4}
```

Comment:
Is there a nightly/latest too?

Reply:
There's a `latest` image yup. Should I add it, and perhaps skip the version metadata tests for that one?

Reply:
So, very good point, because this made me realize we weren't testing against 2.3 either.

=> Updated tox config to test against `2.3` and `latest`.

FTR, testing against 2.3 made me realize two things:

- Users will have to use the admin account to collect metrics on 2.3 (granting permissions on the system tables (even read-only) was only added in 2.3).
- We can only support 2.3.6+ due to a bug in the `server_status` system table: https://github.com/rethinkdb/rethinkdb/issues/6108. (We _could_ work around it by skipping `server_status` based on the detected server version, but I think we should keep the cruft low for now — esp. since 2.3.6 was released in 2017.)